opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js

```diff
@@ -543,14 +543,14 @@ describe('Elasticsearch pagination', () => {
     expect(metaByEntityType['external-reference'].length).toEqual(7);
     expect(metaByEntityType['object-marking'].length).toEqual(21);
     expect(metaByEntityType['kill-chain-phase'].length).toEqual(3);
-    expect(data.edges.length).toEqual(161);
+    expect(data.edges.length).toEqual(160);
```

Comment:
Can you explain why?

Reply:
// 150 instead of 151 before the merge (now 160 instead of 161) because of a relationship 'has-role' that is not counted anymore (has also been removed in the count of internal-relationships)

Reply:
I think its due to a missing default group with default role.